src/lib/swapper/swappers/ThorchainSwapper/utils/addSlippageToMemo.ts

```diff
@@ -2,49 +2,41 @@ import type { ChainId } from '@shapeshiftoss/caip'
 import { BigNumber, bn } from 'lib/bignumber/bignumber'
 import { subtractBasisPointAmount } from 'state/slices/tradeQuoteSlice/utils'
 
-import { DEFAULT_STREAMING_NUM_SWAPS, LIMIT_PART_DELIMITER, MEMO_PART_DELIMITER } from './constants'
+import { MEMO_PART_DELIMITER } from './constants'
 import { assertIsValidMemo } from './makeSwapMemo/assertIsValidMemo'
 
 export const addSlippageToMemo = ({
   expectedAmountOutThorBaseUnit,
-  affiliateFeesThorBaseUnit,
   quotedMemo,
   slippageBps,
   isStreaming,
   chainId,
   affiliateBps,
-  streamingInterval,
 }: {
   expectedAmountOutThorBaseUnit: string
-  affiliateFeesThorBaseUnit: string
   quotedMemo: string | undefined
   slippageBps: BigNumber.Value
   chainId: ChainId
   affiliateBps: string
   isStreaming: boolean
-  streamingInterval: number
 }) => {
   if (!quotedMemo) throw new Error('no memo provided')
 
+  // always use TC auto stream quote (0 limit = 5bps - 50bps, sometimes up to 100bps)
+  // see: https://discord.com/channels/838986635756044328/1166265575941619742/1166500062101250100
+  if (isStreaming) return quotedMemo
+
   // the missing element is the original limit with (optional, missing) streaming parameters
   const [prefix, pool, address, , affiliate, memoAffiliateBps] =
     quotedMemo.split(MEMO_PART_DELIMITER)
 
   const limitWithManualSlippage = subtractBasisPointAmount(
-    bn(expectedAmountOutThorBaseUnit)
-      .minus(affiliateFeesThorBaseUnit)
```

Comment:
affiliate fees handled on the expected amount out on get quote

Reply:
Yeah now that we return the proper memo for limit, we should be much safer WRT regular swaps! Can always revisit and iterate on later if we still see regular swaps failing at limit